jekyll/_cci2/testing-llm-enabled-applications-through-evaluations.adoc

```diff
@@ -42,64 +37,123 @@ Evaluations can cover many aspects of a model performance, including:
 * Respond in a fair and unbiased way, and within a specific style
 * Avoid certain expressions
 
-== Automating evaluations with CircleCI
+Some important differences between evaluations and standard software tests to consider include the following:
+
+* LLMs are predominantly non-deterministic, leading to flaky evaluations, more so than typical deterministic software tests.
+* Evaluation results are subjective. Small regressions in a metric might not necessarily be a cause for concern, unlike failing tests in standard software testing.
+* Evaluation results are manually reviewed to determine whether AI application performance meets expectations since the output metric values can fluctuate.
 
-Evaluations can be expressed as classic software tests, typically characterised by the "input, expected output, assertion" format, and as such they can be automated into CircleCI pipelines.
+Using an open source library or third-party tools can simplify defining evaluations, tracking progress, and reviewing evaluation results.
 
-Two important differences between evals and classic software tests to keep in mind:
+== Automate evaluations with CircleCI
 
-* LLMs are predominantly non-deterministic, leading to flaky evaluations, unlike deterministic software tests.
-* Evaluation results are subjective. Small regressions in a metric might not necessarily be a cause for concern, unlike failing tests in regular software testing.
+*Evaluations can be automated in CircleCI pipelines.* CircleCI enables you to automate decisions regarding the following:
 
-With CircleCI, you can define, automate, and run evaluations using your preferred evaluation framework. Through declaring the necessary commands in your `config.yml`, you can ensure these evaluations are run within your CircleCI pipeline.
+* Acceptance of proposed AI application changes.
+* Deployment and release your AI application.
 
-Using an open source library or third-party tools can simplify defining evaluations, tracking progress, and reviewing the evaluation outcomes.
+With CircleCI, you can define, automate, run evaluations and evaluation testing using your preferred evaluation framework. Through declaring the necessary commands in your pipeline configuration you can ensure evaluations are run and evaluation results are tested within your CircleCI pipeline.
 
-== The CircleCI Evals orb
+You can automate your entire process of running evaluations, reviewing evaluation results and determining whether results meet expectations, removing manual work involved with triggering evaluations and reviewing evaluation results.
+
+=== The CircleCI evals orb
 
 [NOTE]
 ====
-The link:https://circleci.com/developer/orbs/orb/circleci/evals[official Evals orb] supports integrations with Braintrust and LangSmith.
+The link:https://circleci.com/developer/orbs/orb/circleci/evals[CircleCI evals orb] supports integrations with Braintrust and LangSmith.
```

Comment:
I think we should remove this: the 2.0 version of the Orb does not support 3rd-party integrations yet.